src/main/java/com/adzerk/sdk/AdditionalOptions.java

```diff
@@ -44,6 +46,24 @@ public AdditionalOptions apiKey(String apiKey) {
     return this;
   }
 
+  public String[] getDesiredAds() {
+    return this.desiredAds;
+  }
+
+  public AdditionalOptions desiredAds(String desiredAds[]) {
```

Comment:
Should be a `List<String>` instead of a fixed array.

Reply:
Updated